source/Jacwright/RestServer/RestServer.php

```diff
@@ -358,7 +358,9 @@ public function getMethod()
 			$method = 'PUT';
 		} elseif ($method == 'POST' && strtoupper($override) == 'DELETE') {
 			$method = 'DELETE';
-		}
+		} elseif ($method == 'PATCH' && strtoupper($override) == 'PATCH') {
+            $method = 'PATCH';
+        }

```

Comment:
This block is either unnecessary, or should be fixed to use POST. The override here allows a POST to pretend to be a DELETE. For example, when clicking a button in the browser that says "Remove Item From Cart", that button could be in a `<form method="post">` element. This basically allows you to bypass firewalls that reject calls that are not GET or POST. But it may be unnecessary these days.

Either way, this is a no-op. If the method is already "PATCH"  no need to set it to "PATCH" again.